src/server/routes/csv.js

```diff
@@ -124,10 +123,13 @@ router.post('/meters', validateMetersCsvUploadParams, async (req, res) => {
 		}
 
 		const conn = getConnection();
-		await uploadMeters(req, res, csvFilepath, conn);
-		success(req, res, 'Successfully inserted the meters.');
+		const newlyCreatedMeters = await uploadMeters(req, res, csvFilepath, conn);
+		// Send the newly created meters back to the frontend
+		res.status(200).send({ message: '<h1>SUCCESS</h1>Successfully inserted the meters.', meters: newlyCreatedMeters });
```

Comment:
See a longer discussion in src/server/test/web/csvPipelineTest.js. I think this returned message changed too. It also changed how it is sent.